server/users_test.go

```diff
@@ -305,7 +305,7 @@ func TestService_NewUser(t *testing.T) {
 			},
 			wantStatus:      http.StatusUnauthorized,
 			wantContentType: "application/json",
-			wantBody:        `{"code":401,"message":"Cannot set SuperAdmin"}`,
+			wantBody:        `{"code":401,"message":"Cannot change the status of a SuperAdmin"}`,
```

Comment:
when i received this error in the client, it made me think that it's actually not possible to change the status of a superAdmin, which isn't true; rather, i lacked the authorization to do so. it was a 401 Unauthorized, but i wonder if the message would more clearly be `Insufficient authorization to change the status of a SuperAdmin` ? i also got this on a `Create` when, as an Admin, i tried to create a user with `superAdmin: true`, and the error message didn't seem to quite fit the bill there either